backend/api/tests/views/conftest.py

```diff
@@ -28,7 +32,14 @@ def api_client() -> test.APIClient:
 
 @pytest.fixture
 def create_compute_task():
-    def _create_compute_task(compute_plan, n_data_sample=4):
+    def _create_compute_task(
+        compute_plan: Optional[ComputePlan] = None,
+        n_data_sample: int = 4,
```

Comment:
I know it's a port from the previous code, but any idea why this default value? (4)